src/controllers/DefaultController.php

```diff
@@ -8,6 +8,7 @@ public function handle(Request $request, $db)
 
         // just add the available methods, with links
         $retval['events']          = $request->base . '/' . $request->version . '/events';
+        $retval['categories']      = $request->base . '/' . $request->version . '/categories';
```

Comment:
I'm happy to remove the entry and resubmit the PR if you'd like. On the other hand, do we have a place for a developer to go who wants to integrate w/ joind.in and needs to know what APIs are available?